layouts/partials/scripts.html

```diff
@@ -12,6 +12,7 @@
 {{ $s = $s | append (resources.Get "js/chooser.js") }}
 {{ $s = $s | append (resources.Get "js/noselect.js") }}
 {{ $s = $s | append (resources.Get "js/tracking.js") }}
+{{ $s = $s | append (resources.Get "js/docs-feedback.js") }}
```

Comment:
+1 to adding something like Stencil.js for things like these going forward. (Happy to look into that myself.)